### PR TITLE
Add CRMBot connector

### PR DESCRIPTION
**Pull request by michal-turzynski**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: certified-connectors/CRMBot/readme.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Create powerful conversation experience for your customers right from your CRM system. Give users new ways to interact with your company by building engaging voice and text-based conversational interfaces, such as voice apps and chatbots, powered by AI. Connect with users on your website, mobile app, the Google Assistant, Amazon Alexa, Facebook Messenger, and other popular platforms and devices. And do it directly from your existing CRM implementation.
```
Comment (dougelliott) [inline certified-connectors/CRMBot/readme.md:1]: Could this section be given a title? Even just the name of the connector is fine. Thanks in advance!

File: certified-connectors/CRMBot/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Create powerful conversation experience for your customers right from your CRM system. Give users new ways to interact with your company by building engaging voice and text-based conversational interfaces, such as voice apps and chatbots, powered by AI. Connect with users on your website, mobile app, the Google Assistant, Amazon Alexa, Facebook Messenger, and other popular platforms and devices. And do it directly from your existing CRM implementation. 
+
+## Prerequisites
+
+Our clients have to undergo a registration process (currently offered through MS Teams chatbot as well as our website/sales bot. This provides them with a secret API key which is necessary to use our connector.
```
Comment (dougelliott) [inline certified-connectors/CRMBot/readme.md:5]: There appears to be a missing ending parenthesis here for the opening one right before "...currently offered through"

**Review by dougelliott — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Comment by dougelliott** [comment]
@michal-turzynski Checking in on things. Any questions we could help answer as far as the requested changes?

File: certified-connectors/CRMBot/readme.md (hunk 1 above)
Comment (michal-turzynski) [reply certified-connectors/CRMBot/readme.md]: Should be resolved now. Thank you! @dougelliott

File: certified-connectors/CRMBot/readme.md (hunk 2 above)
Comment (michal-turzynski) [reply certified-connectors/CRMBot/readme.md]: Also resolved cc @dougelliott

**Review by dougelliott — APPROVED** [review]

**Pull request merged**